recipes/oscars/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "oscars" %}
+{% set version = "2.1.8" %}
+{% set sha256 = "c37853089db601537c48484ee72b84ae40893daf0d4b86030e4bf080dc3965b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dhidas/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [unix]
```

Comment:
Pretty sure the recipe needs a compiler on Windows as well...

Reply:
Yes, I agree. I don't remember exactly why I excluded it in https://github.com/nsls-ii-forge/oscars-feedstock.
@kbeeperez, let's get the compiler for all systems unconditionally.

I think the selector can be removed next to the `mpi4py` dependency too.